site.yml

```diff
@@ -74,9 +74,8 @@ asciidoc:
 #   ocis
     latest-ocis-version: 'next'
     previous-ocis-version: 'next'
-    ocis-version: '2.0.0-beta1'
+    ocis-version: '2.0.0-beta3'
```

Comment:
I thought that we are calling this beta.3 now?
@mmattel

Reply:
🤦‍♂️

Reply:
I made PR #435